ios/Podfile.lock

```diff
@@ -246,4 +246,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 91efd4097cc41f032416d67ecaac21cc2de8dd55
 
-COCOAPODS: 1.11.3
+COCOAPODS: 1.11.2
```

Comment:
Why we need to downgrade this?

Reply:
Oh. It is spawned automatically when run on iOS 16. It's not necessary. I will remove it.